source/_components/sensor.netatmo_public.markdown

```diff
@@ -0,0 +1,40 @@
+---
+layout: page
+title: "Netatmo Public"
+description: "Instructions on how to add sensors using Netatmo public data to Home Assistant."
+date: 2018-07-22 21:18
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: netatmo.png
+ha_category: Sensor
+ha_release: 0.75
+---
+
+The Netatmo Public sensor allows you to expose data from [Netatmo](https://weathermap.netatmo.com/) to Home Assistant, even if you don't have a netatmo device yourself.
+
+To enable the Netatmo sensor, you have to set up [netatmo](/components/netatmo/).
+
+Then, enable the sensor by adding the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: netatmo_public
+    areas:
+      - lat_ne: 40.719
+        lon_ne: -73.735
+        lat_sw: 40.552
+        lon_sw: -74.105
+```
+
+Configuration variables:
```

Comment:
Please use the configuration tags, for more information please see:
https://home-assistant.io/developers/documentation/create_page/#configuration

Reply:
Thanks, updated that. Looks like I missed that part in the docs, and the example I looked at wasn't using that tag.